guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,436 @@
+
+Example workload(s):
+
+* [Scheduled Radiology Diagnostics](#scheduled-radiology-diagnostics)
+
+### Batch Computational Workload {#batch-computational-workload}
+
+Work is enqueued for future processing when a request is received. Response to the request indicates if the work was successfully enqueued.
+
+Work starts with a parent-job, when the scheduler determines. The parent-job inspects the input dataset and generates child-jobs to process it. Each child-job is submitted via a new request. Child-jobs will be [asynchronous computational workloads](#asynchronous-computational-workload).
+
+As each of the child-jobs completes, the callback address provided as part of the initial request is used to send a completion notification. Once all of the child-jobs have completed, the callback address provided as part of the initial request is used to send a final completion notification.
+
+Example workload(s):
+
+* Single patient DICOM series sent to multiple algorithms for prospective diagnosis.
+* [Retrospective Study](#retrospective-study): cohort study of 10K patients sent to a single algorithm.
+
+### Streaming Computational Workload {#streaming-computational-workload}
+
+Work starts when a request is received. If the system cannot process the request immediately, an error is returned. Input is provided continuously as part of the request. Output is returned continuously as part of the response. The request completes when either end “hangs up” or disconnects.
+
+Example workload(s):
+
+* Image enhancement and overlay for ultrasound.
+
+## Clinical Capacity Planning {#clinical-capacity-planning}
+
+The clinical capacity needed for a typical large hospital or hospital system is used in determining the estimated computing workload seen at the system level.  The computing required will be allocated in one of the following ways based on the [clinical use cases](#clinical-use-cases).
+
+1. Reserved capacity (always on) for QA type of use cases
+2. Shared capacity (for emergency and scheduled radiology)
+3. On-demand / R&D capacity for research workloads
+
+The estimated number of machines per modality for a large hospital will vary significantly so  instead, the number of studies per modality per month data will be used to estimate a workload description for capacity planning.  
+
+The estimated number of radiology exams per modality is provided from a few sources including data provided from three large hospitals across the US and in the UK as well as data from the UK National Health Service Diagnostic Information Dataset (DID) on RIS. 
+
+Here are the modalities derived from the data sets in descending order of average number of studies per month.
+| Modality                      | Other Terminology |
+| ----------------------------- | ----------------- |
+| Plain Radiography             | XR, X-Ray         |
+| Diagnostic Ultrasonography    | US, Ultrasound    |
+| Computerized Axial Tomography | CT, CAT           |
```

Comment:
```suggestion
| Computed Tomography | CT, CAT           |
```
This terminology is more standard these days

Reply:
egads
Fixed